frontend/app-development/layout/AppBar/AppBar.tsx

```diff
@@ -62,14 +62,15 @@ export const AppBar = ({
         </ul>
         <div className={classes.rightContent}>
           <div className={classes.rightContentButtons}>
-            <Button
+            {process.env.NODE_ENV !== 'production' &&
```

Comment:
This will work, but will also hide the button in dev.altinn.studio (since we run production-build for frontend when we deploy). Probably not a big issue since the button doesn't go anywhere at the moment.